sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -978,37 +978,41 @@ object JdbcUtils extends Logging with SQLConfHelper {
       options: JDBCOptions,
       namespace: String,
       comment: String): Unit = {
+    val statement = conn.createStatement
+    try {
+      statement.setQueryTimeout(options.queryTimeout)
+      val dialect = JdbcDialects.get(options.url)
+      dialect.createSchema(statement, namespace, comment)
+    } finally {
+      statement.close()
+    }
+  }
+
+  def namespaceExists(conn: Connection, options: JDBCOptions, namespace: String): Boolean = {
```

Comment:
since we are touching this file, can we rename `namespace` to `schema` for all these methods in this file?

Reply:
OK